fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -545,8 +546,11 @@ private Query getFilterQuery(Set<ApiFilter> filters, int perPage) {
         TreeSet<DimensionRow> filteredDimRows;
         int documentCount;
         initializeIndexSearcher();
+        RequestLog.startTiming("LuceneReadLock");
         lock.readLock().lock();
+        RequestLog.stopTiming("LuceneReadLock");
```

Comment:
We shouldn't time this. A read-lock should be fast enough that the timings will actually be much slower than the lock. If we _really_ want to get a sense of how much time this takes, we should run a profiler on this running under an actual workload.